### ISSUE TITLE
`/directory` filters

### ISSUE THREAD

**Issue by ymollard** [BDX-town/Mangane#271]
J'ai le sentiment que les 2 filtres de [bdx.town/directory](https://bdx.town/directory) sont cassés, non ?
- Lorsque "uniquement Bdx.town" est actif, le bouton "charger plus" charge un JSON vide
- Lorsque "Les ptits nouveaux en 1er" est actif, bah les premiers sont pas les petits nouveaux 😆

**Comment by Cl0v1s** [comment]
I have the felling that's more a combo of long lasting requests vs bad-ui / no feedback

* For "only instance" -> you can only see the accounts that authorized to appear there. We should hide the "load more" button if no data are available
* For "new users first" -> what is making you saying that ? You compared the date of creation for the accounts that appeared at top ?

**Comment by Cl0v1s** [comment]
It's probably that, but anyway you're right that particular screen is lacking some loader and more indications !

**Comment by Cl0v1s** [comment]
The issue was indeed an non-clear loading state and desync between fetch state and what's shown in the UI

**Comment by Cl0v1s** [comment]
fixed in https://github.com/BDX-town/Mangane/pull/281

**Comment by ymollard** [comment]
I think that the button "Load more" should still be replaced with a message "No more result" if 100% of profiles have been loaded.

**Comment by Dehelssey** [comment]
or just not be displayed if useless^^
